libs/host/Configuration/Options.cs

```diff
@@ -639,6 +643,7 @@ public GarnetServerOptions GetServerOptions(ILogger logger = null)
                 CheckpointThrottleFlushDelayMs = CheckpointThrottleFlushDelayMs,
                 EnableScatterGatherGet = EnableScatterGatherGet.GetValueOrDefault(),
                 ReplicaSyncDelayMs = ReplicaSyncDelayMs,
+                ReplicaMaxLag = ReplicaMaxLag,
```

Comment:
ReplicationOffsetMaxLag? Else it is not clear that the lag is expressed in terms of offset number.

Reply:
Renamed it to ReplicationOffsetMaxLag!